idaes/apps/grid_integration/bidder.py

```diff
@@ -755,8 +759,8 @@ def write_results(self, path):
             None
         """
 
-        print("")
-        print("Saving bidding results to disk...")
+        _logger.info("\n")
```

Comment:
You do not need the new line with the logger. Also, in the next line, I would use only a single period (`.`) instead of an elipsis (`...`).